src/main/java/uk/gov/companieshouse/officer/delta/processor/model/DeltaIdentification.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.companieshouse.officer.delta.processor.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+import uk.gov.companieshouse.api.appointment.Identification;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DeltaIdentification {
```

Comment:
Good spot @cgregoryCH! I had forgotten we had split the API and persistence models :)

Reply:
To make it even more confusing, this is actually the model which we deserialise the incoming delta into so we can then convert it to the api request model. We then do the opposite like you mentioned in company-appointments.api.ch.gov.uk.